### PR TITLE
[benchmark] add docker files for benchmark

### PR DESCRIPTION
**Pull request by 0-o-0**
## Motivation

Add docker files for benchmark binary, just like what have for validator, client, etc.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Deployed on my test cluster

### CODE REVIEW

File: docker/bench/bench_init.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+echo "$MINT_KEY" | base64 -d > /opt/libra/etc/mint.key
+/opt/libra/bin/ruben -a $AC_HOST -f /opt/libra/etc/mint.key -d $AC_DEBUG --metrics_server_address "0.0.0.0:14297" $@
```
Comment (littlepretty) [inline docker/bench/bench_init.sh]: Just a note: after this PR merged, is it safe to delete the `-d` option from RuBen and also the `-d $AC_DEBUG` here?

**Comment by fbandersnatch** [comment]
Is this a copy of the other file we use now?

**Comment by 0-o-0** [comment]
@fbandersnatch Yes, it's a copy with some path changes. Does it look good to you or do you have other plans on migrating the docker files?

File: docker/bench/bench_init.sh (hunk 1 above)
Comment (0-o-0) [reply docker/bench/bench_init.sh]: good point, removed `-d`

**Comment by fbandersnatch** [comment]
Since we're moving this here: can you update the other images to all have an identical base, and then resubmit the PR to make CircleCI happy

**Pull request merged**